bucket/biblioteq.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "2020.09.09",
+    "description": "Professional archiving, cataloging, and library management suite providing connectivity to PostgreSQL and SQLite.",
+    "homepage": "https://textbrowser.github.io/biblioteq/",
+    "license": "BSD",
```

Comment:
Invalid SPDX identifier

Reply:
Ok will fix, it's the bsd 3-clause license.

Reply:
SPDX identifier BSD-3-Clause now added.